libraries/AC_AutoTune/AC_AutoTune.h

```diff
@@ -32,13 +71,15 @@ class AC_AutoTune {
     virtual void run();
 
     // save gained, called on disarm
-    void save_tuning_gains();
+    // made virtual for to add multi/heli specific save routines
```

Comment:
no need for the comment about why it was made virtual I think.  That'll all be in the past soon

Reply:
done. Thanks!